### PR TITLE
Feature/heatmap

### PR DESCRIPTION
**Pull request by mdflynn**
### Description

- Adds heatmap to sightings map
- Adds production api into app

### Closes issue(s)

#20 #21 

### Testing
n/a
### Screenshots
n/a
### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### Checklist
- [ ] I have written tests for this code (happy & sad)
- [ ] I have updated the Readme
- [ ] I ran full test suite - all tests passing 
- [x] Lint code

### CODE REVIEW

File: src/Components/SightingsMap/SightingsMap.scss [hunk 1]
```diff
@@ -2,3 +2,20 @@
   height: 400px;
   width: 400px;
 }
+
```
Comment (peytonjo) [inline src/Components/SightingsMap/SightingsMap.scss:5]: Styling may need some more work

**Review by peytonjo — APPROVED** [review]
Nice work

File: src/Components/SightingsMap/SightingsMap.js [hunk 2]
```diff
@@ -62,42 +83,71 @@ const SightingsMap = () => {
...
-          </div>
-        </InfoWindow>
-      )}
-    </GoogleMap>
+          </InfoWindow>
+        )}
+      </GoogleMap>
+      <div className="button-div">
+        <button onClick={toggleHeatMap} className="main-button">Toggle HeatMap</button>
+        {/* <button>Show Recent Sightings</button> */}
```
Comment (jdxsmith) [inline src/Components/SightingsMap/SightingsMap.js:145]: Is this leftover code or is this going to be implemented in the future?

File: src/api.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import {
 
 const fetchSightings = (dispatch) => {
   dispatch(fetchSightingsPending());
-  fetch("http://localhost:5000/api/v1/reports")
+  fetch("https://ancient-mesa-60922.herokuapp.com/api/v1/reports")
```
Comment (jdxsmith) [inline src/api.js:10]: Glad you were able to get the backend incorporated!

**Review by jdxsmith — APPROVED** [review]
The heatmap is super cool, great addition!

**Pull request merged**